common/rand.go

```diff
@@ -46,18 +46,15 @@ func (r *Rand) GetFr() (fr.Element, error) {
 	}
 }
 
-// TODO: this function is temporary. It returns a big.Int
-// from some random 128 bytes. It should be better defined
-// regarding the underlying group order.
-func (r *Rand) GetBigInt128() (big.Int, error) {
-	var byts [128]byte
-	if _, err := r.rand.Read(byts[:]); err != nil {
-		return big.Int{}, fmt.Errorf("get randomness: %s", err)
-
+func (r *Rand) GetFrBigInt() (big.Int, error) {
+	var randElem fr.Element
+	if _, err := randElem.SetRandom(); err != nil {
+		return big.Int{}, fmt.Errorf("get random GT: %s", err)
 	}
```

Comment:
I think we could also do: `randElem, err := GetFr()` to avoid some repetition. NBD anyway.